modules/installation-configuration-parameters.adoc

```diff
@@ -79,34 +71,6 @@ endif::[]
 ifeval::["{context}" == "installing-aws-customizations"]
 :aws:
 endif::[]
-ifeval::["{context}" == "installing-openstack-installer-custom"]
-:osp:
-:osp-custom:
-endif::[]
-ifeval::["{context}" == "installing-openstack-installer-kuryr"]
-:osp:
-:osp-kuryr:
-endif::[]
-ifeval::["{context}" == "installing-openstack-user"]
-:osp:
-:osp-custom:
-endif::[]
-ifeval::["{context}" == "installing-openstack-user-kuryr"]
-:osp:
-:osp-kuryr:
-endif::[]
-ifeval::["{context}" == "installing-openstack-user-sr-iov"]
-:osp:
-:osp-custom:
-endif::[]
-ifeval::["{context}" == "installing-openstack-user-sr-iov-kuryr"]
-:osp:
-:osp-kuryr:
-endif::[]
-ifeval::["{context}" == "installing-openstack-installer-restricted"]
-:osp:
-:osp-custom:
-endif::[]
```

Comment:
Hey @maxwelldb I wanted to make you aware of a refactor I am applying to installation docs to make sure you are good with the update in your content area. For more information on the why, please see [osdocs-6493](https://issues.redhat.com/browse/OSDOCS-6493). Also for a finished example, you can checkout https://github.com/openshift/openshift-docs/pull/64049.

First - are you OK with me applying this update to OpenStack? If so, my question is directly related to the `osp-custom` and `osp-kuryr` attributes. While defined, the only attribute I see applied to any of the content in this module is `osp`. I am nearly certain that defining `osp` at line 111 will continue to render all content correctly for the refactor, but I just wanted to check in with you.

Reply:
I'm fine with this if it works! 

`osp-custom` and `osp-kuryr` might be entirely vestigial in this case. It's hard for me to say for sure, as the installation docs conditionals story has been so rough--particularly in the long-long ago. 📆 

What is the plan to test this change? Historically, conditionals have been the cause of a number of silent docs bugs. @mjpytlak

Reply:
> What is the plan to test this change? Historically, conditionals have been the cause of a number of silent docs bugs

The verification is simple. Because I am simply reusing `osp` in a new `ifeval` directive. I do a compare of the current published docs against the content in the new assembly.

Reply:
Seems copacetic to me.